wpilibc/src/main/native/cppcs/RobotBase.cpp

```diff
@@ -117,11 +118,6 @@ bool RobotBase::IsNewDataAvailable() const { return m_ds.IsNewControlData(); }
 std::thread::id RobotBase::GetThreadId() { return m_threadId; }
 
 RobotBase::RobotBase() : m_ds(DriverStation::GetInstance()) {
-  if (!HAL_Initialize(500, 0)) {
-    wpi::errs() << "FATAL ERROR: HAL could not be initialized\n";
-    wpi::errs().flush();
-    std::terminate();
-  }
```

Comment:
Is this safe to remove?

Reply:
Yes. RunHALInitialization gets called as the first think when the robot is started, before the robot class is initialized. 

https://github.com/wpilibsuite/allwpilib/blob/340c1198035e1f2df67803821a4ae564bd4745e4/wpilibc/src/main/native/cppcs/RobotBase.cpp#L34